samples/MiniZinc/data.dzn

```diff
@@ -0,0 +1 @@
+n=3;
```

Comment:
These are samples written by me specifically to avoid any issue with licenses. There are lots of larger example programs available under MIT/BSD-style licenses, but I thought it would be cleaner to just contribute original code rather than having to duplicate the license/attribution here.